policy/scan/local_scanner.go

```diff
@@ -52,6 +52,7 @@ type LocalScanner struct {
 	// allows setting the upstream credentials from a job
 	allowJobCredentials bool
 	disableProgressBar  bool
+	reportType          ReportType
```

Comment:
We should consider adjusting the enum https://github.com/mondoohq/cnspec/blob/5ea86601d8542258804f3d8c19493f653b4feb19/policy/scan/scan.proto#L34-L38 and make 0 Unspecified. This would ensure that full needs to be explicitly set.